src/app/src/main/java/com/example/bookmark/models/Geolocation.java

```diff
@@ -1,10 +1,12 @@
 package com.example.bookmark.models;
 
+import java.io.Serializable;
+
 /**
  * TODO: Description of class.
  * @author Kyle Hennig.
  */
-public class Geolocation {
+public class Geolocation implements Serializable {
```

Comment:
@kylehennig I added this to Geolocation because I assume you'll be using this but let me know if you were going to implement serializability another way

Reply:
Yup. I don't use the Serializable interface, so this won't affect me. Instead, I use the FirestoreSerializable interface.